2018/proposal.md

```diff
@@ -0,0 +1,90 @@
+PyCon AU 2018 Education Seminar Proposal
+========================================
+
+If accepted, the PyCon Australia Education Seminar will bring together educators who are using Python
+in teaching or looking for new ways to use coding in education. The goal is to share stories, strategies
+and techniques for engaging students and teaching with Python that can be directly applied by other educators. 
+
+Python has long had reputation as an effective beginner programming language. Bringing together educators and
+developers helps developers provide better support for beginners in Python and its libraries, as well as
+giving educators the rare opportunity to develop their own technical skills further.
+
+As in previous years, talk topics at the miniconf will be open to any educational use of Python but the core
+focus will continue to be on primary and secondary school education. This includes both formal classroom
+instruction and extracurricular activities.
+
+However, some changes and additions are also being proposed based on our experiences in previous iterations
+of the event.
+
+Event Organisers
+----------------
+
+* Lead organiser: Nick Coghlan
+  * 2015 Education Seminar founder & coordinator, 2016/17 co-coordinator with Katie Bell
+  * CPython core developer
+  * Former member of the Python Software Foundation Board of Directors (2014-16)
+
+* Co-organiser: Nicky Ringland? (Bruce Fuda?, Amanda Hogan?)
+  * Australian Computing Academy: https://aca.edu.au/about.html
+
+Event Outline
+-------------
+
+Prior to the afternoon tea break, the 2018 Education Seminar will follow the same
+structure as in previous years: an invited speaker prior to morning tea, 3 speakers
+from the Call for Proposals prior to lunch, then 3 more speakers prior to afternoon
+tea.
+
+For the final session, we'd like to work with the main conference organisers to
+arrange to run a "Student Showcase", inspired by the well-received student
+presentations at PyCon Australia 2017.
+
+The general idea of this part of the proposal would be to:
+
+* Provide a way for teachers & senior secondary students to nominate as potential
+  presenters at the Showcase (similar to the way Gungahlin College students did
+  through the regular CfP for 2017)
+* Accept up to 6 projects for presentation (these would each be given a 15 minute
+  presentation slot *without* any question & answer time, and a ~5 minute break
```

Comment:
I think it'd be good for students to get one question. Proposal: we have one question for each project prepared, and use it, or a question from twitter if there's a good one. I think having a bit of dialog with the students is useful, and answering questions is scary, so giving them a chance to do so in a supportive environment is well worth giving them only 12 or 13 minutes to present.

Reply:
Ah, moderated questions only - I like it.

Reply:
That way we can reassure the kids that we'll only ask nice questions (i.e. things that they will already know) if they're worried about that sort of thing, and not like: "Why didn't you use <this framework you've never heard of>?" or similar.